ui/components/app/flask/icons/npm-icon.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function NpmIcon({ className, color }) {
+  return (
+    <svg
+      className={className}
+      width="25"
+      height="10"
+      viewBox="0 0 25 10"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <path
```

Comment:
Font Awesome already has NPM icon build in, and we're using it.
https://fontawesome.com/v5.15/icons/npm?style=brands

You can replace NpmIcon with
`<i class="fab fa-npm"></i>`

Reply:
Good catch, thanks!

Reply:
Does this mean you are going to remove this component @hmalik88 ?

Reply:
@georgewrmarshall I'm not sure, I'm inclined to keep it. The npm font-awesome icon doesn't work when I try to use it. I think it may be that specific icon that doesn't work, because I tried others and they worked.

Reply:
Found the issue, yes I will be removing.